docs/interactive_engine/deployment.md

```diff
@@ -74,10 +74,11 @@ deployment and management of applications. To deploy GIE standalone using Helm,
    kubectl describe svc [YOUR_RELEASE_NAME]-gie-standalone-frontend \
    | grep "Endpoints:" | awk -F' ' '{print $2}'
    ```
-   You should see the GIE Frontend service endpoint as `<ip>:<gremlinPort>`.
+   You should see two exposed endpoints for GIE Frontend service, one is `<ip>:<gremlinPort>` for gremlin querying, another is `<ip>:<cypherPort>` for cypher querying.
```

Comment:
Explicitly use two commands, one for obtaining Cypher endpoint, the other for Gremlin endpoint.

Reply:
done